### ISSUE TITLE
render fails: questionable use of here() in analysis-steps.qmd

### ISSUE THREAD

**Issue by vjcitn** [lmweber/PrinciplesSTA#47]
```
processing file: analysis-steps.qmd
1/13                  
2/13 [unnamed-chunk-1]
Error in gzfile(file, mode) : cannot open the connection
Calls: .main ... eval_with_user_handlers -> eval -> eval -> saveRDS -> gzfile

Quitting from lines 22-31 [unnamed-chunk-1] (analysis-steps.qmd)

ERROR: Unexpected end of JSON input
1/12 packages newly attached/loaded, see sessionInfo() for details.
```

line 30 of analysis-steps.qmd expects an "outputs" folder for 'here', but
in a fresh checkout that won't exist.   I made a little progress by hardcoding a
folder path but there are many uses of here in the code.

Maybe here() works fine when building in Rstudio (I see an Rproj file at root)
but I don't know how to do that.

**Comment by lmweber** [comment]
Thanks for this feedback, good point. I think I created the `outputs/` directory manually and then assumed it exists, which is not necessarily true. I reproduced the error just now by deleting all my output directories, and had the same problem.

I'll try fixing this by adding a line to create the `outputs/` directory if it does not exist.

I have been building from the command line with `quarto render` or `quarto publish`, so there should be no other problems due to the use of `here()` with or without RStudio. However if this isn't correct, then it may be better if I remove the use of `here()` altogether.
